README.md

```diff
@@ -1,12 +1,60 @@
 # Cybersecuirty Suite for Forensic Analysis
-UNR CS 425/426 Project By Alvin Leung, Kameron Bettridge, Raymond Pai, and Zachary Wilhite
-
-Libraries needed:
-nltk
-os
-pandas
-pickle
-random
-math
-shutil
-tkinter
+UNR CS 425/426 Project
```

Comment:
This is how you can add a comment to a specific part of the code.

I will add a more in depth description later of the CSFA.

Reply:
After reading, you can then resolve the conversation when either the comment is read, or you reach a consensus on whatever change was suggested or whatever